rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -1962,6 +1962,36 @@ void getPendingFederatorPublicKeyOfType_withSpecificRskAndMstKeys_returnsFederat
         }
     }
 
+    @Test
+    void getNewFederationBtcUTXOs_whenNoUTXOsWereSaved_returnsEmptyList() {
+        storageAccessor = new InMemoryStorage();
```

Comment:
This is not needed since this is being set in the `setup` method.

Reply:
you right, thanks

Reply:
nope, this test is out of the `PendingFederationTestsWithNonNullFederation` class